lib/orogen/templates/config/Base.cmake

```diff
@@ -1,8 +1,10 @@
 include(OrogenPkgCheckModules)
+if(NOT TARGET regen)
```

Comment:
The two ifs (this one and the one about check-uptodate) are broken. Remove them (especially since, I believe, they are not needed anymore)